docker/develop/Dockerfile

```diff
@@ -136,9 +136,9 @@ RUN git clone https://github.com/jtv/libpqxx /tmp/libpqxx; \
 
 # install cpp_redis with tacopie
 RUN git clone https://github.com/Cylix/cpp_redis /tmp/cpp_redis; \
-    (cd /tmp/cpp_redis ; git checkout f390eef447a62dcb6da288fb1e91f25f8a9b838c); \
+    (cd /tmp/cpp_redis ; git checkout f390eef447a62dcb6da288fb1e91f25f8a9b838c); \ # v4.3.0
```

Comment:
it does not work like this.
`\` must be the very last token on the line

put comment before the line

Reply:
Done